src/mudlet.qrc

```diff
@@ -178,7 +178,9 @@
         <file>mudlet-lua/lua/generic-mapper/generic_mapper.xml</file>
         <file>mudlet-lua/lua/utf8_filenames.lua</file>
         <file>mudlet-mapper.xml</file>
+        <file>Mudlet_splashscreen_development.png</file>
         <file>Mudlet_splashscreen_main.png</file>
+        <file>Mudlet_splashscreen_ptb.png</file>
```

Comment:
The `Mudlet` text rendering doesn't look as good as what we have on the main one - could that be improved?

Also, the sunlight spilling into the arch - could you lower it just a tad?

I think this will make it look even better.

Reply:
That was supposed to be a MOON not a Sun... yeah, I'll see if I can turn it down a bit! :sweat_smile:

Reply:
I'll have another go - I see now that the process I used to extract the original logo text uses a binary yes/no to select the pixels that made up the text - but that falls over when the text is anti-aliased as it is here. Thorsten did get back to me and he confirmed what you said that the text was constructed by hand - he express surprise that we did not have the .svg file for it but the only file of that type we have is just for the arch itself - NOT for the text.  I have explained this in a reply but have not had another response yet.  Unless you happen to have it tucked away somewhere?

Reply:
We do have the svg's! You should have asked. See all of the svg's in the https://github.com/Mudlet/artwork repo :)